src/polkitbackend/polkitbackendauthority.h

```diff
@@ -52,6 +52,14 @@ struct _PolkitBackendAuthority
   GObject parent_instance;
 };
 
+enum
+{
+	LOG_LEVEL_ERROR,	// log errors only
```

Comment:
Even if other levels are not used, please add support for all syslog levels, so that we can add LogControl1 support later: https://www.freedesktop.org/software/systemd/man/latest/org.freedesktop.LogControl1.html (I can wire that up once this is merged)

Reply:
Also, the exact same levels as in syslog should be used. This is because those priorities will be propagated to the entries in the journal, and ideally there is a clear mapping between what is visible here in the code and what the journal allows. Doing a 1:1 mapping is the most reasonable.